tests/ci/README.md

```diff
@@ -8,7 +8,8 @@ We use underscore connected strings in a pipeline yaml file name to indicate the
 <compute>_<test>_<os>_<environment>_<branch>.yml
 ```
 
-For example, if a unit test for Spark utility functions of master branch is run on an Linux Azure Data Science Virtual Machine, it will be named as 
+For example, if a unit test for Spark utility functions of master branch is run on an Linux Azure Data Science Virtual Machine, it will be named as:
+
 ```
 dsvm_unit_linux_pyspark_master.yml
```

Comment:
Maybe rename this one too.

Reply:
all the tests are going to be replaced on a subsequent PR, including the naming of this